draft-ietf-quic-http.md

```diff
@@ -1318,6 +1318,10 @@ connection to become idle, or MAY initiate an immediate closure of the
 connection.  An endpoint that completes a graceful shutdown SHOULD use the
 HTTP_NO_ERROR code when closing the connection.
 
+If a client has consumed all available bidirectional stream IDs with requests,
```

Comment:
This is true, but would the HTTP layer know this?

Reply:
It has to be tracking the greatest stream ID it has received a request on in order to put that value in a GOAWAY frame.  So, yes.  Another implementation might be to retrieve this number and then find that it's out-of-bounds, but it amounts to the same thing.